### ISSUE TITLE
requirements.txt is missing in Pypi's sdist archive

### ISSUE THREAD

**Issue by onkelbeh** [pywbem/pywbem#2150]
Hi,
is there a reason for not having requirements.txt in the sdist tar.gz at Pypi?

I made a [Gentoo ebuild](https://github.com/onkelbeh/HomeAssistantRepository/tree/master/dev-python/pywbem), which compiles without problems with the tar.gz here from github, which includes the requirements.txt. If I take the archive from Pypi (where the requirements.txt is missing), compilation fails:

```
>>> Compiling source in /var/tmp/portage/dev-python/pywbem-0.16.0/work/pywbem-0.16.0 ...
 * python3_7: running distutils-r1_run_phase distutils-r1_python_compile
python3.7 setup.py build -j 10
Traceback (most recent call last):
  File "setup.py", line 72, in <module>
    requirements = get_requirements('requirements.txt')
  File "setup.py", line 52, in get_requirements
    with open(requirements_file, 'r') as fp:
FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
 * ERROR: dev-python/pywbem-0.16.0::HomeAssistantRepository failed (compile phase):
```
These 2 archives should be identical or at least contain the same, I would prefer using an archiv from pypi in general, because this has certain advantages in Gentoo's build system.

**Comment by andy-maier** [comment]
There is no reason for not including it. I'll look into it.

**Comment by andy-maier** [comment]
Addressed by PR #2152 in master, and TBD in 0.17.1.

**Comment by andy-maier** [comment]
Once PR #2152 succeeds, can you please try your build using its branch (`andy/enable-setup-install`), and post here whether it worked for you?

**Comment by onkelbeh** [comment]
Patch https://github.com/pywbem/pywbem/pull/2155 looks good to me, but still is not merged yet.

**Comment by andy-maier** [comment]
Thanks for the review. We'll merge it once Karl (my co-maintainer) approves it, and when the other currently open PRs targeting 0.17.1 are also reviewed and merged, I'll release 0.17.1. Should take a day or two, I think.

**Comment by andy-maier** [comment]
0.17.1 has its release PR up for review and should be released in the next day or so.
